helpers/auth/validator.py

```diff
@@ -17,4 +18,11 @@ def assert_wing_is_required(office, kwargs):
 
 def verify_email(email):
     return bool(re.match('^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$',
-                email))
+                         email))
+
+
+class ErrorHandler():
+
+    def check_conflict(self, entity, entity_name):
```

Comment:
Looks good but only fixes one error, check request above for how to handle more errors.